### PR TITLE
faq-first

### PR DESCRIPTION
**Pull request by bharatgupta77**
Developed a structure for a FAQ.

### CODE REVIEW

File: faq.html [hunk 1]
```diff
@@ -3,9 +3,40 @@
...
+				<li>Members : React</li>
+				<li>Events: React</li>
+				<li>Main App : Ember</li>
+			</ul>
+			 
+			       
+
+			</h3>
+
+	</section>
```
Comment (ankushdharkar) [inline faq.html:30]: Indentation is all spaced up. Let's fix that please

File: css/FAQ.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
+h1{
+	text-align:center;
+}
+
+.headingalignment{
+	margin-left: 5%;
+}
+.sectionalignment{
+	margin-left: 10%;
```
Comment (ankushdharkar) [inline css/FAQ.css:9]: Let's use more descriptive names? "about-us", "questions-group".

What if in future we have more headings, we will not be able to use any names there.

File: faq.html [hunk 3]
```diff
@@ -3,9 +3,40 @@
...
+
+	<h2 class="headingalignment">Git Related Questions </h2>
+	<section class="sectionalignment">
```
Comment (ankushdharkar) [inline faq.html:33]: Let's rename these too

File: faq.html [hunk 4]
```diff
@@ -3,9 +3,40 @@
...
+		   
+		   <h3>Question 1 - I don't know GIT. Can I still contribute? </h3>
+		   <h3>Answer - </h4>
```
Comment (ankushdharkar) [inline faq.html:36]: Started h3, but closed h4 - Mismatch

File: faq.html (hunk 1 above)
Comment (bharatgupta77) [reply faq.html:30]: Fixed.

File: faq.html (hunk 4 above)
Comment (bharatgupta77) [reply faq.html:36]: Fixed

File: css/FAQ.css (hunk 2 above)
Comment (bharatgupta77) [reply css/FAQ.css:9]: topic and question-group. Will it work ?

File: faq.html [hunk 5]
```diff
@@ -1,11 +1,54 @@
...
 </head>
+
 <body>
 
+	<h1> FAQ </h1>
+
+	<h2 class="topic">About RDS </h2> 
+	<section class="questions-group">
+			<h3>Question - What is RDS? </h3>
+			<h3>Answer  - </h4>
```
Comment (ankushdharkar) [inline faq.html:20]: h3 opened, but closed with h4

**Pull request merged**